lib/src/tabs/Tab.tsx

```diff
@@ -0,0 +1,181 @@
+import React, { forwardRef, Ref } from "react";
+import styled from "styled-components";
+import { TabProps } from "./types";
+import DxcBadge from "../badge/Badge";
+
+const Tab = forwardRef(
+  (
+    { active, tab, tabIndex, hasLabelAndIcon, iconPosition, onClick, onMouseEnter, onMouseLeave }: TabProps,
+    ref: Ref<HTMLButtonElement>
+  ): JSX.Element => {
+    return (
+      <TabContainer
+        role="tab"
+        type="button"
+        tabIndex={tabIndex}
+        disabled={tab.isDisabled}
+        aria-disabled={tab.isDisabled}
+        aria-selected={active}
+        hasLabelAndIcon={hasLabelAndIcon}
+        iconPosition={iconPosition}
+        ref={ref}
+        onClick={() => {
+          onClick();
+        }}
+        onMouseEnter={() => {
+          onMouseEnter();
+        }}
+        onMouseLeave={() => {
+          onMouseLeave();
+        }}
+      >
+        <MainLabelContainer
+          hasBadge={tab.notificationNumber}
+          hasLabelAndIcon={hasLabelAndIcon}
+          iconPosition={iconPosition}
+        >
+          {tab.icon && (
+            <TabIconContainer hasLabelAndIcon={hasLabelAndIcon} iconPosition={iconPosition}>
+              {typeof tab.icon === "string" ? <TabIcon src={tab.icon} /> : tab.icon}
+            </TabIconContainer>
+          )}
+          <LabelTextContainer>{tab.label}</LabelTextContainer>
+        </MainLabelContainer>
```

Comment:
I think here we can use `Typography` component for the label and give there the font styles, color, letterSpacing... instead of in the `TabContainer`. We can also remove the `LabelTextContainer` since it is an empty one and we do not need it.

Reply:
Done!